Index: settings.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>rootProject.name = 'shift'\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/settings.gradle b/settings.gradle
--- a/settings.gradle	(revision 9f1225265e0a64ce014dc80aaf883bdabad9a045)
+++ b/settings.gradle	(date 1733604794574)
@@ -1,2 +1,4 @@
 rootProject.name = 'shift'
+include 'task4'
+include 'task4'
 
